tutorials/06-physics-simulation-concepts.md

```diff
@@ -0,0 +1,154 @@
+\page physicsconcepts Ignition Physics simulation concepts
+
+This tutorial introduces simulation concepts that are used in Ignition Physics.
+As the SDF model is used to describe simulation scenarios in Ignition, the
+Ignition Physics simulation concepts are described closely related to the SDF model.
+
+## Physics simulation versus animation
+
+Dynamics simulators for robotics is more challenging than the ones used for
+animating virtual characters due to these aspects:
+- The forces, torques, frictions, etc. are not crucial in animation since the
+law of physics can be violated.
+- Real-time requirements and physical reality can be less constraining for
+purely visual display.
+- Several challenges for simulating complex physical structures, such as
+humanoid robot:
+  - Numerical instability in computing inverse kinematics, fluid flows, etc.
+    prevents correct physics simulation.
+  - Contact forces between bodies are hard to model due to their discontinuity
+    characteristics, especially in the soft body case, may result in unrealistic
+    contacts or physically unfeasible contact forces.
+
+Ignition adopts SDF structure to describe not only the visual element but also
```

Comment:
I recommend `SDFormat` instead of `SDF`, since there are some other acronyms that use `SDF`.